nautobot_golden_config/views.py

```diff
@@ -421,6 +421,8 @@ def diff_structured_data(backup_data, intended_data):
             output = config_details.backup_config
         elif config_type == "intended":
             output = config_details.intended_config
+        elif config_type == "config_to_push":
+            output = get_config_to_push(config_details.intended_config, request.user, device)
```

Comment:
In thinking about it, I think the signature needs to be something like: 
`get_config_to_push(config_details.intended_config, config_details.backup_config, config_details.missing_config, config_details.extra_config, request.user, device)`

To account for future use cases. Perhaps the config I want to push is just part of the missing configuration?

Reply:
aren't the missing_config and extra_config extracted from the other two?
For the backup, I see the point if we need to apply some remediation stuff.

Reply:
Continue convo in other comment.